src/dataset-builder/DatasetBuilder.ts

```diff
@@ -644,11 +645,12 @@ export const DatasetBuilderContents = ({
       ]),
       requestValid &&
         h(ActionBar, {
-          prompt: h(Fragment, [
+          prompt: div({ style: { display: 'flex' } }, [
             datasetRequestParticipantCount.status === 'Ready'
-              ? datasetRequestParticipantCount.state.result.total
+              ? DisplayParticipantCount(datasetRequestParticipantCount.state.result.total)
```

Comment:
```suggestion
              ? h(DisplayParticipantCount, { count: datasetRequestParticipantCount.state.result.total })
```

And then you also need to do a change to the signature which I'll add below.